src/mongocxx/options/find.hpp

```diff
@@ -30,7 +31,6 @@ namespace options {
 /// Class representing the optional arguments to a MongoDB query.
 ///
 class MONGOCXX_API find {
-
```

Comment:
Add this back please - we try to add a blank line between the class open and the leading public.